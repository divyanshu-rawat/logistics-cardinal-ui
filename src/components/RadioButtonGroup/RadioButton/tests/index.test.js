import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import RadioButtonConsumer from '..';

const renderedComponent = () =>
  shallow(<RadioButtonConsumer label="foo" id="foo" value="foo" />);

describe('RadioButtonConsumer', () => {
  it('should render the component', () => {
    const renderComponent = renderedComponent({});
    expect(toJson(renderComponent)).toMatchSnapshot();
  });
});
