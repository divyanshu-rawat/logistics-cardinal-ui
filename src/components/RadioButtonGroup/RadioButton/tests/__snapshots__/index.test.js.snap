// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButtonConsumer should render the component 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
